apps/archive/__init__.py

```diff
@@ -89,9 +89,11 @@ def init_app(app):
     register_component(ItemAutosave(app))
     register_model(ItemAutosaveModel(EveProxy(superdesk.get_backend())))
 
+    superdesk.privilege(name='monitoring_view', label='Can access monitoring view', description='')
     superdesk.privilege(name='archive', label='Archive', description='User can view the published content.')
     superdesk.privilege(name='ingest', label='Ingest', description='User can view content in ingest and fetch it.')
-    superdesk.privilege(name='spike', label='Spike', description='User can spike content.')
+    superdesk.privilege(name='spike_read', label='Can see spiked items', description='User can see spiked content.')
+    superdesk.privilege(name='spike', label='Can spike items', description='User can spike content.')
```

Comment:
These labels are more of a description then a label. This would mean the `Spike` role would now sorted with other roles starting with `C`.

We should be displaying the description of privileges in the UI and leave the names short, i.e. `Spike` and show `User can spike content` on hover (or another column in the table, as we have the sapce).

Reply:
We don't currently display a description in the UI and we want to ship this feature fast so I'll leave it as is for now. We are thinking about revamping the entire privileges section in the future so privileges are grouped. That would be a good opportunity to take a look at naming as well. I don't see it being clear if we name privileges spike1, spike2 and explain it in the description. If we can describe a privilege clearly in a few words, say 6 for example, I'd put it in the name a leave a description blank if there's not anything else to add.

Reply:
In Analytics and Planning we use short labels and descriptive descriptions.
i.e.
label: Planning - Spike Event Items
description: Ability to spike an Event

When alphabetically sorted, this privilege is placed with other Planning privileges starting with 'S'.

Adding one more column in the privileges table would be a very quick and easy way to show the description of a Privilege, while still have a short label/name of the privilege.

The label should be short and the description should describe what it is for. Such as:
label: Spike
description: User can spike items

label: Spike - View
description: Use can see spiked items

Later on when refactoring we can add the Superdesk module or group by any other means. But simply adding in a column in the UI to show the description should be enough for now.

Reply:
I mostly agree with you, except regarding the quickness to add it to the UI. It's quick technically to make the text appear, but it [doesn't look good](https://user-images.githubusercontent.com/2076953/65593523-f0380a80-df90-11e9-8562-0678fab0339e.png). There are also duplicate data that needs to be cleaned up.

Reply:
what about using [tooltips](https://ui-framework.superdesk.org/#/components/tooltips) ?

Reply:
I've added tooltips in the [client PR](https://github.com/superdesk/superdesk-client-core/pull/3118)

![screenshot_9](https://user-images.githubusercontent.com/2076953/65610127-2d13f980-dfb1-11e9-89c7-6987e11b51dc.png)

Reply:
The tooltips are truncated if the description is too long. But I guess this will do for now